src/freedv_api.c

```diff
@@ -859,7 +859,7 @@ int freedv_bits_to_speech(struct freedv *f, short speech_out[], short demod_in[]
     int decode_speech = 0;
     if ((rx_status & FREEDV_RX_SYNC) == 0) {
 
-        if (f->squelch_en == 0) {
+        if (!f->squelch_en) {
 
```

Comment:
I haven't used `bool` much, so just did some reading and confirmed `if (!bool_variable)` always means "if variable false". Makes sense.  I was curious as I am unsure what value (0 or 1) is used for `true` internally.